Verifier.ts

```diff
@@ -19,57 +19,47 @@ export class Verifier<T extends Payload> extends Actor<Verifier<T>> {
 		} else
 			this.algorithms = undefined
 	}
-	async verify(token: string | Token | undefined, ...audience: string[]): Promise<T | undefined> {
-		let result: Payload | undefined = undefined
-		if (token) {
-			const splitted = token.split(".", 3)
-			if (splitted.length == 3) {
-				try {
-					const oldDecoder = token.includes("/") || token.includes("+") // For backwards compatibility.
-					const header: Header = JSON.parse(
-						new TextDecoder().decode(Base64.decode(splitted[0], oldDecoder ? "standard" : "url"))
-					)
-					if (await this.verifySignature(header, splitted)) {
-						result = JSON.parse(
-							new TextDecoder().decode(Base64.decode(splitted[1], oldDecoder ? "standard" : "url"))
-						) as Payload
-					}
-				} catch {
-					// Keep result undefined
-				}
-				result = result && this.verifyAudience(result.aud, audience) ? result : undefined
-				if (result) {
-					const now = Verifier.now
-					if (result?.iat && result.iat > 1000000000000)
-						result.iat = Math.floor(result.iat / 1000)
-					if (result?.exp && result.exp > 1000000000000)
-						result.exp = Math.floor(result.exp / 1000)
-					result =
-						!this.algorithms ||
-						((result.exp == undefined || result.exp > now) && (result.iat == undefined || result.iat <= now + 60))
-							? result
-							: undefined
-				}
-				if (result) {
-					result.token = token
-					result = await this.transformers.reduceRight(async (p, c) => c.reverse(await p), Promise.resolve(result))
-				}
+	private async decode(
+		token: string | Token | undefined
+	): Promise<{ header: Header; payload: Payload; signature: string; splitted: [string, string, string] } | undefined> {
+		let result: Awaited<ReturnType<Verifier<T>["decode"]>>
+		const splitted = token?.split(".", 3)
+		if (splitted?.length != 3)
+			result = undefined
+		else {
+			try {
+				const standard: cryptly.Base64.Standard = token?.match(/[/+]/) ? "standard" : "url"
+				const decoder = new cryptly.TextDecoder()
+				const header: Header = JSON.parse(decoder.decode(cryptly.Base64.decode(splitted[0], standard)))
+				const payload: Payload = JSON.parse(decoder.decode(cryptly.Base64.decode(splitted[1], standard)))
+				if (payload.iat && payload.iat > 1000000000000)
+					payload.iat = Math.floor(payload.iat / 1000)
+				if (payload.exp && payload.exp > 1000000000000)
+					payload.exp = Math.floor(payload.exp / 1000)
```

Comment:
These where used for backwards compatibility with a very early version that contained timestamps in milliseconds. We might not need this anymore.